File: Iniciante/1013 - O Maior/Python/1013.py
def main():
    values_str = input().split()
    values_num = list(map(int, values_str))
    larger_number = higherValue(values_num)
    print(f"{larger_number} eh o maior")


def higherValue(values):
    higher = max(values)
    return higher


if __name__ == "__main__":
    main()

File: Iniciante/1013 - O Maior/JavaScript/1013.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const [a, b, c] = input.split(" ").map(Number);
  const largerNumber = higherValue(a, b, c);
  console.log(`${largerNumber} eh o maior`);
}

function higherValue(a, b, c) {
  const higher = Math.max(a, b, c);
  return higher;
}

main();
